src/strategy/strategy.ts

```diff
@@ -23,10 +23,7 @@ export interface Constraint {
 export interface Segment {
   id: number;
   name: string;
```

Comment:
Do we use name anywhere?

Reply:
I can remove it too. I left it in the backend response for now though. But maybe I can remove it too.